### ISSUE TITLE
Unable to download previous releases from download page

### ISSUE THREAD

**Issue by anandparmar** [playframework/playframework.com#78]
On the download page, the links for previous releases contain the link of the latest release not the respective release. for example href for Play 2.5.0 release is "https://downloads.typesafe.com/typesafe-activator/1.3.9/typesafe-activator-1.3.9-minimal.zip" which is of latest release i.e. Play 2.5.1

**Comment by gmethvin** [comment]
I agree this is really confusing. You can use the latest version of activator and start a project from a template for an older version of Play, for example [play-scala-2.4](http://www.lightbend.com/activator/template/play-scala-2.4) using `activator new PROJECTNAME play-scala-2.4`. But we don't really make it clear that this is possible on the download page.

Actually if you want 2.5.0, I think the only way to do it is to take a 2.5 (which will be the latest 2.5.x version) template and manually change the version in `project/plugins.sbt`. But I think this is okay since most of the time you should be using the latest version in that release series.

Anyway, the download page should be updated to at least make it clear how this works. It

**Comment by NjeriNjoroge** [comment]
Hi, my name is Grace i'm an Android Developer. I'm looking forward to participate in Outreachy round 15 internship. Just wanted to know if I could help in this issue and how to start.
